spec/authorization_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Horoshop::Authorization do
+  let(:instance) { double('Instance', username: 'user', password: 'pass', url: 'http://api.horosop.com') }
+
+  before do
+    allow(instance).to receive(:token=)
+    allow(instance).to receive(:expiration_timestamp=)
+    allow(Time).to receive(:now).and_return(Time.now)
```

Comment:
You stubbing Time.now with Time.now ?

Reply:
I agree, it’s not a logical stub; it was originally created for the zone method with a current return.
After application, Time.now was not deleted but simply changed.